src/coord_finder.cpp

```diff
@@ -150,9 +166,17 @@ class CoordFinder {
     }
 
 
-    static std::vector<lidarPoint> createLidarPoints(const std::vector<float>& distances, double startAngle, double angleIncrement) {
+    /**
+    * @brief Creates a vector of LidarPoints
+    * 
+    * @param[in] distances detected by Lidar
+    * @param[in] startAngle - the angle to start at
+    * @param[in] angleIncrement - the amoung to increment the angle for each distance
+    * @returns the lidarPoints vector
```

Comment:
Quick note, I noticed variables are name in different styles. EX some vars are  using underscore while others are using pascal case. I think id prefer all variables to be pascal case. we can discuss this more and decide on casing.

Reply:
changed to lower case